qualibration_graphs/nv_center/quam_config/utils/connectivity_nv_center_interface.py

```diff
@@ -0,0 +1,171 @@
+from qualang_tools.wirer.connectivity.channel_spec import ChannelSpec
```

Comment:
I created a PR in py-qua-tools to handle this file: [PR-306](https://github.com/qua-platform/py-qua-tools/pull/306)